src/main/java/ch/njol/unofficialmonumentamod/mixins/SpriteAtlasHolderAccessor.java

```diff
@@ -0,0 +1,13 @@
+package ch.njol.unofficialmonumentamod.mixins;
+
+import net.minecraft.client.texture.SpriteAtlasHolder;
+import net.minecraft.client.texture.SpriteAtlasTexture;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+//This accessor class won't stay, it's only here to see what broke down
```

Comment:
Is this commend outdated or did you intend to remove the class?

Reply:
I forgot